contracts/ROSCA.sol

```diff
@@ -130,10 +150,47 @@ contract ROSCA {
 
       currentRound++;
       LogStartOfRound(currentRound);
+    } else if (currentRound == membersAddresses.length) {
+        endOfROSCA = true;
     }
   }
 
   /**
+   * try to send the fund to the opt_destination from source account
+   */
+  function sendFund(address source,address opt_destination) internal returns(bool success){
```

Comment:
1. I've used the prefix "opt_" to specify that an argument is optional. Here I don't think it's optional, so can we rename it to "destination".

2. I think the name "source" is misleading because it implies that funds come from it, whereas the funds come from the contract itself. Consider renaming to "participant" and updating the comment accordingly.